packages/client/hmi-client/src/services/workflow.ts

```diff
@@ -807,10 +807,16 @@ export function getActiveOutput(node: WorkflowNode<any>) {
 	return node.outputs.find((o) => o.id === node.active);
 }
 
-export function updateOutputPort(node: WorkflowNode<any>, updatedOutputPort: WorkflowOutput<any>) {
-	let outputPort = node.outputs.find((port) => port.id === updatedOutputPort.id);
-	if (!outputPort) return;
-	outputPort = Object.assign(outputPort, updatedOutputPort);
+/**
+ * Update the output of a node referenced by the output id
+ * @param node
+ * @param updatedOutput
+ */
+export function updateOutput(node: WorkflowNode<any>, updatedOutput: WorkflowOutput<any>) {
+	const foundOutput = node.outputs.find((output) => output.id === updatedOutput.id);
+	if (foundOutput) {
+		Object.assign(foundOutput, updatedOutput);
+	}
```

Comment:
So frankly I don't know why we want to go through the exercise of renaming this - as it had been mentioned several times before, this is an escape hatch, and there is no need nor desire to make it seem like it is a legitimate api-interface.

But if you really want to do this, please also update the workflow specification to reflect this change
https://docs.google.com/document/d/1mHEeViL5TpA90p2XtheWvkKR8ofcvieDdi4wWIIkbv8/edit

Reply:
Done